pytorch_lightning/trainer/trainer.py

```diff
@@ -1674,41 +1672,41 @@ def _on_exception(self) -> None:
 
     @property
     def accelerator(self) -> Accelerator:
-        return self.training_type_plugin.accelerator
+        return self.strategy.accelerator
 
     @property
-    def training_type_plugin(self) -> TrainingTypePlugin:
-        return self._accelerator_connector.training_type_plugin
+    def strategy(self) -> Strategy:
+        return self._accelerator_connector.strategy
```

Comment:
Here we need to add the deprecation path, reroute the old to the new + test in tests/deprecated_api

Reply:
Oh totally forgot about this is trainer API, will update